dev/com.ibm.ws.logging/src/com/ibm/ws/logging/source/LogSource.java

```diff
@@ -27,7 +25,6 @@
 import com.ibm.ws.logging.data.LogTraceData;
 import com.ibm.ws.logging.internal.WsLogRecord;
 import com.ibm.ws.logging.utils.LogFormatUtils;
-import com.ibm.ws.logging.utils.SequenceNumber;
```

Comment:
The raw long sequence number still needs to be grabbed in LogSource, and set in the LogData record.  We want the records to be numbered when the LogData events are published, rather than at formatting time.